### PR TITLE
Contacts 0.05: Nicer UI, Refactoring, new icon

### PR DESCRIPTION
**Pull request by devsnd**

### CODE REVIEW

File: apps/contacts/contacts.app.js [hunk 1]
```diff
@@ -17,35 +17,49 @@ function callNumber (number) {
...
     menu['No Contacts'] = () => {};
-  } else {
-    for (const e of wp) {
-      const closureE = e;
-      menu[e.name] = () => showContact(closureE);
-    }
   }
-  menu["Add"] = addContact;
-  menu["Remove"] = removeContact;
+  contacts.map((e, idx) => {
```
Comment (bobrippling) [inline apps/contacts/contacts.app.js:32]: You can use `.forEach` if you don't use the result of `map`, so we avoid creating a new array :)

```suggestion
  contacts.forEach((e, idx) => {
```

**Review by bobrippling — COMMENTED** [review]
Looks good, I'll give it a test soon (there's one linter problem outstanding too)

**Comment by devsnd** [comment]
Thanks @bobrippling ! I'll try to fix it asap, but it would be a Christmas miracle if I make it today ;)

Thanks for your suggestions and happy Christmas holidays :)

**Comment by devsnd** [comment]
Oh cool, I didn't realize I could just fix it on the phone in the browser...! The linter caught a mistake I made during refactoring, so that check was well worth it. It should be all ready now 🤞

**Comment by bobrippling** [comment]
Nice job, thanks for the quick fixes!

**Pull request merged**